content/en/project/easy-guide-on-python-packages-utilisation/index.md

```diff
@@ -1,38 +1,38 @@
 ---
 type: "project" # DON'T TOUCH THIS ! :)
-date: "2020-05-16" # Date you first upload your project.
+date: "2022-07-05" # Date you first upload your project.
 # Title of your project (we like creative title)
-title: "This is an example project page which serves as a template"
+title: "An easy guide to "not throwing your expensive computer out the window because you can't run a Python neuroimaging tool"
```

Comment:
```suggestion
title: "An easy guide to \"not throwing your expensive computer out the window because you can't run a Python neuroimaging tool\""
```